ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.19.yaml

```diff
@@ -130,6 +130,58 @@ tests:
           cpu: 1000m
           memory: 512Mi
     workflow: optional-operators-ci-aws
+- always_run: false
+  as: e2e-test-hcp-aws
+  optional: true
+  run_if_changed: .+\.go|go\.(\bmod\b|\bsum\b)|Makefile|Dockerfile|.dockerignore|.*\.ya{0,1}ml|build/.+
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OO_INDEX: ci-index
+    env:
+      OO_CHANNEL: stable
+      OO_INSTALL_NAMESPACE: openshift-adp
+      OO_PACKAGE: oadp-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: e2e
+      cli: latest
+      commands: make TEST_HCP=true test-e2e
+      credentials:
+      - mount_path: /var/run/oadp-credentials
+        name: oadp-credentials
+        namespace: test-credentials
+      from: test-oadp-operator-1.5
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 512Mi
+    workflow: optional-operators-ci-aws
+- as: e2e-test-hcp-aws-periodic
+  optional: true
+  steps:
```

Comment:
I think periodic doesn't work as an optional.. it doesn't need to be because it doesnt block PRs.

When you ran make command it probably removed the cron

This is what you had
```
- as: e2e-test-hcp-aws-periodic
  cron: 0 10 * * *
  steps:
```

Reply:
Not at all, I've removed the cron once we want to make it optional because both options are not compatible together. I can recover the cron if you want.